client/blocks/plugins-scheduled-updates-multisite/schedule-form.tsx

```diff
@@ -51,7 +53,11 @@ export const ScheduleForm = ( { onNavBack, scheduleForEdit }: Props ) => {
 
 	const translate = useTranslate();
 
-	const { data: sites } = useSiteExcerptsQuery( [ 'atomic' ] );
+	const siteFilter = ( site: SiteExcerptData ): boolean => {
+		return ( site as SiteDetails ).capabilities?.update_plugins;
+	};
+
+	const { data: sites } = useSiteExcerptsQuery( [ 'atomic', 'capabilities' ], siteFilter );
```

Comment:
Shouldn't this be `	const { data: sites } = useSiteExcerptsQuery(['atomic'], siteFilter, 'all', [ 'capabilities' ]) ; `?